src/plugins/auctionHouseModule/createBid.ts

```diff
@@ -17,12 +47,30 @@ export type CreateBidOperation = Operation<
 >;
 
 export type CreateBidInput = {
+  auctionHouse: AuctionHouse;
+  wallet?: PublicKey | Signer; // Default: identity
+  authority?: PublicKey | Signer; // Default: auctionHouse.authority
+  mintAccount: PublicKey; // Required for checking Metadata
+  tokenAccount?: PublicKey; // Default: ATA
+  price?: SolAmount | SplTokenAmount; // Default: 0 SOLs or tokens.
+  tokens?: SplTokenAmount; // Default: token(1)
+  bookkeeper?: Signer; // Default: identity
+  printReceipt?: boolean; // Default: true
+
   // Options.
   confirmOptions?: ConfirmOptions;
 };
 
 export type CreateBidOutput = {
   response: SendAndConfirmTransactionResponse;
+  buyerTradeState: Pda;
+  tokenAccount: PublicKey;
```

Comment:
For public bids, this could be null.

```suggestion
  tokenAccount: Option<PublicKey>;
```